en/docs/apis/challenge-question-rest-api.md

```diff
@@ -0,0 +1,40 @@
+---
+template: templates/swagger.html
+---
+
+# User's Challenge Questions and Answers API Definition - v1
+
+!!! Note 
+    Do the following to try out the REST APIs with your local instance of WSO2 Identity Server. 
```

Comment:
Shall we replace this with the statement below. I feel it's better and consistent with the other docs as well. 
Follow the steps given below to try out the REST APIs with your local instance of WSO2 Identity Server (WSO2 IS).